server/routes/api/fileOperations.ts

```diff
@@ -13,7 +13,7 @@ import pagination from "./middlewares/pagination";
 
 const router = new Router();
 
-router.post("fileOperations.info", auth(), async (ctx) => {
+router.post("fileOperations.info", auth({ admin: true }), async (ctx) => {
```

Comment:
Note: These routes are already authorized via policies, but this is another "layer" of security by restricting the role so early on.